app/routes/__frontend/resources/developer-resources/open-source/about-open-source.mdx

```diff
@@ -57,8 +57,12 @@ Working out how to get started can be the most challenging part. So we've compil
 
 **Virtual Coffee's open source resources:**
 
-- <Link to="/resources/developer-resources/open-source/contributor-guide">Contributor Guide</Link> if you want to contribute to an open source project.
-- <Link to="/resources/developer-resources/open-source/maintainer-guide">Maintainer Guide</Link> if you're going to maintain your open source project.
+- <Link to="/resources/developer-resources/open-source/contributor-guide">
+  	Contributor Guide
+  </Link> if you want to contribute to an open source project.
```

Comment:
@danieltott I forgot to look here 😂
So prettier format is what making it not working.
When I reverse it to the previous format, it works just fine.

Anyway, I think it doesn't matter if I revert the commit and fix it. Because then, whenever I push it, prettier action will auto format it this way.

Reply:
Let me see if I can work around it. Maybe minor tweak to the format of the sentence can resolve the problem.